widgetssdk/src/main/java/com/glia/widgets/launcher/ConfigurationManager.kt

```diff
@@ -27,7 +25,7 @@ internal interface ConfigurationManager {
      * [queueIds] A list of queue IDs to be used for the engagement launcher.
      * When null, the default queues will be used.
```

Comment:
Oh, seems I missed to change the doc here to `When empty, the default queues will be used.`
Could you please change?